src/components/box/box.component.tsx

```diff
@@ -55,103 +59,73 @@ export interface BoxProps
-
-const calculatePosition = (props: Omit<PositionProps, "zIndex">) => {
-  const { position, ...rest } = positionFn(props);
-
-  return {
-    position,
-    zIndex: ["sticky", "fixed"].includes(position) ? 1 : undefined,
-    ...rest,
-  };
-};
-
-export const Box = styled.div<BoxProps>`
-  ${() => {
-    if (!isDeprecationWarningTriggered) {
-      isDeprecationWarningTriggered = true;
-      Logger.deprecate(
-        "Previous props that could be spread to the `Box` component are being removed. Only props documented in the intefaces will be supported."
-      );
-    }
-    return css``;
-  }}
-
-  ${space}
-  ${layout}
-  ${flexbox}
-  ${calculatePosition}
-
-  ${({ theme, borderRadius = "borderRadius000" }) =>
-    !theme.roundedCornersOptOut &&
-    css`
-      border-radius: var(--${borderRadius});
-    `}
-
-  ${({ color, bg, backgroundColor, ...rest }) =>
-    styledColor({ color, bg, backgroundColor, ...rest })}
-
-  ${({ overflowWrap }) =>
-    overflowWrap &&
-    css`
-      overflow-wrap: ${overflowWrap};
-    `}
-
-  ${({ scrollVariant }) =>
-    scrollVariant &&
-    css`
-      scrollbar-color: ${boxConfig[scrollVariant].thumb}
-        ${boxConfig[scrollVariant].track};
-
-      &::-webkit-scrollbar {
-        width: 8px;
-      }
-      &::-webkit-scrollbar-track {
-        background-color: ${boxConfig[scrollVariant].track};
-      }
-      &::-webkit-scrollbar-thumb {
-        background-color: ${boxConfig[scrollVariant].thumb};
-      }
-    `}
-
-  ${({ boxSizing }) =>
-    boxSizing &&
-    css`
-      box-sizing: ${boxSizing};
-    `}
-
-    ${({ display, gap, columnGap, rowGap }) =>
-    (display === "flex" || display === "inline-flex") &&
-    css`
-      ${gap !== undefined &&
-      css`
-        gap: ${boxConfig.gap(gap)};
-      `}
-
-      ${columnGap !== undefined &&
-      css`
-        column-gap: ${boxConfig.gap(columnGap)};
-      `}
-
-      ${rowGap !== undefined &&
-      css`
-        row-gap: ${boxConfig.gap(rowGap)};
-      `}
-    `};
-
-  ${({ boxShadow }) =>
-    boxShadow &&
-    css`
-      box-shadow: var(--${boxShadow});
-    `}
-`;
-
-Box.defaultProps = {
-  theme: BaseTheme,
-};
+export const Box = React.forwardRef<HTMLDivElement, BoxProps>(
+  (
+    {
+      "data-component": dataComponent,
+      id,
+      role,
+      overflowWrap,
+      scrollVariant,
+      boxSizing,
+      tabIndex,
+      gap,
+      columnGap,
+      rowGap,
+      className,
+      children,
+      bg,
+      backgroundColor,
+      boxShadow,
+      borderRadius,
+      color,
+      opacity,
+      ...rest
+    }: BoxProps,
+    ref
+  ) => {
+    return (
+      <StyledBox
+        id={id}
+        role={role}
+        overflowWrap={overflowWrap}
+        scrollVariant={scrollVariant}
+        boxSizing={boxSizing}
+        gap={gap}
+        columnGap={columnGap}
+        rowGap={rowGap}
+        tabIndex={tabIndex}
+        className={className}
+        ref={ref}
+        bg={bg}
```

Comment:
**suggestion:** Do we need the `bg` and `backgroundColor` props? I was under the impression that `bg` was just shorthand for `backgroundColor`, and it appears that is still the case. I've done some digging and found the `styledColor` util which accepts either `bg` or `backgroundColor` I'm assuming (possibly incorrectly) due to `bg` or `backgroundColor` being used interchangeably across the codebase. Imo we just need one or the other here

Reply:
I know this wasn't introduced by you, but might be worth investigating to see if we can remove one

Reply:
It's a good suggestion and I think we will streamline this as part of the styled-system removal at a later date. For now supporting both (since it was possible before), will remain